CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+### 0.1.7
```

Comment:
We have not released 0.1.6 yet, so you can move this bug fix to the 0.1.6 entry. Thanks!

Reply:
Thanks, @aaalaniz. I've fixed this.